packages/wasm-dpp/src/state_transition/validation/validate_state_transition_identity_signature.rs

```diff
@@ -43,12 +45,14 @@ impl StValidator {
 pub async fn validate_state_transition_identity_signature_wasm(
     external_state_repository: ExternalStateRepositoryLike,
     js_state_transition: &JsValue,
+    execution_context: &StateTransitionExecutionContextWasm,
     bls_adapter: JsBlsAdapter,
 ) -> Result<ValidationResultWasm, JsValue> {
     let bls = BlsAdapter(bls_adapter);
     let state_repository = Arc::new(ExternalStateRepositoryLikeWrapper::new(
         external_state_repository,
     ));
+    let execution_context: StateTransitionExecutionContext = execution_context.to_owned().into();
```

Comment:
I'm not sure why this is needed - you're passing references to it everywhere anyway? `&execution_context`

Reply:
SDK does not need it indeed.
I think we will get rid of that in the next iteration, or make it optional at least for wasm